aquadoggo/src/db/models/operation.rs

```diff
@@ -65,14 +65,20 @@ pub struct OperationFieldsJoinedRow {
     pub schema_id: String,
 
     /// The previous operations of this operation concatenated into string format with `_` seperator.
-    pub previous_operations: String,
+    pub previous_operations: Option<String>,
 
     /// The name of this field.
-    pub name: String,
+    ///
+    /// This is an Option as a delete operation contains no fields.
+    pub name: Option<String>,
```

Comment:
Also `name` `field_type` and `value` are options (to account for delete operations)